azure-functions-maven-plugin/src/test/java/com/microsoft/azure/maven/function/invoker/storage/EventHubProcesser.java

```diff
@@ -0,0 +1,152 @@
+
+    private static final String SAS_KAY_NAME = "RootManageSharedAccessKey";
+
+    private EventHubNamespace eventHubNamespace;
+    private StorageAccount storageAccount;
+    private ResourceGroup resourceGroup;
+
+    private ExecutorService executorService = Executors.newCachedThreadPool();
+    private Map<String, EventHubClient> eventHubClientMap = new HashMap<>();
+
+    public EventHubProcesser(String resourceGroupName, String namespaceName, String storageAccountName)
+            throws Exception {
+        final Azure azureClint = CommonUtils.getAzureClient();
+
+        if (azureClint.resourceGroups().contain(resourceGroupName)) {
+            resourceGroup = azureClint.resourceGroups().getByName(resourceGroupName);
+        } else {
+            resourceGroup = azureClint.resourceGroups().define(resourceGroupName).withRegion(Region.US_EAST).create();
+        }
+
+        final boolean isEventHubNamespaceExist = azureClint.eventHubNamespaces().list().stream()
+                .anyMatch(namespace -> namespace.name().equals(namespaceName) &&
+                        namespace.resourceGroupName().equals(resourceGroupName));
+        if (isEventHubNamespaceExist) {
+            eventHubNamespace = azureClint.eventHubNamespaces().getByResourceGroup(resourceGroupName, namespaceName);
+        } else {
+            eventHubNamespace = azureClint.eventHubNamespaces()
+                    .define(namespaceName).withRegion(resourceGroup.region())
+                    .withExistingResourceGroup(resourceGroupName).create();
+        }
+
+        storageAccount = azureClint.storageAccounts().getByResourceGroup(resourceGroupName, storageAccountName);
+        if (storageAccount == null) {
+            storageAccount = azureClint.storageAccounts()
+                    .define(storageAccountName).withRegion(resourceGroup.region())
+                    .withExistingResourceGroup(resourceGroup)
+                    .withSku(StorageAccountSkuType.STANDARD_LRS)
+                    .withGeneralPurposeAccountKindV2().create();
+        }
+    }
+
+    public EventHub createOrGetEventHubByName(String eventHubName) throws Exception {
+        final Azure azureClient = CommonUtils.getAzureClient();
+        final List<EventHub> resultList = eventHubNamespace.listEventHubs().stream()
+                .filter(eventHub -> eventHub.name().equals(eventHubName)).collect(Collectors.toList());
+        if (resultList.size() == 0) {
+            return azureClient.eventHubs().define(eventHubName).withExistingNamespace(eventHubNamespace)
+                    .withExistingStorageAccountForCapturedData(storageAccount, eventHubName)
+                    .withDataCaptureEnabled().create();
+        } else {
+            return resultList.get(0);
+        }
+    }
+
+    public void sendMessageToEventHub(String eventhub, String message) throws Exception {
+        final EventHubClient ehClient = getEventHubClientByName(eventhub);
+        final String payload = message;
```

Comment:
What's the point?

Reply:
Follow by Azure Document: https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-java-get-started-send#send-events